.env.example

```diff
@@ -52,4 +52,5 @@ MATTERS_STRIPE_CONNECT_CLIENT_ID=
 MATTERS_SLACK_TOKEN=
 MATTERS_SLACK_PAYOUT_CHANNEL=
 MATTERS_SLACK_STRIPE_ALERT_CHANNEL=
+MATTERS_SLACK_QUEUE_CHANNEL=
```

Comment:
Please make sure it's added to ElasticBeanstalk.

Reply:
I checked you added it, right?